### PR TITLE
Parser for file /etc/rabbitmq/rabbitmq-env.conf

### PR DESCRIPTION
**Pull request by psachin**
Also included docs & doctest

Signed-off-by: Sachin Patil <psachin@redhat.com>

### CODE REVIEW

File: insights/parsers/rabbitmq.py [hunk 1]
```diff
@@ -192,3 +206,44 @@ def parse_content(self, content):
                         "auto_delete should be true or false: {0}".format(line))
             else:
                 raise ParseException("Data appears invalid: {0}".format(line))
+
+
+@parser(Specs.rabbitmq_env)
+class RabbitMQEnv(SysconfigOptions):
+    """Parse the content of file ``/etc/rabbitmq/rabbitmq-env.conf`` using
+    the ``Parser`` base class.
```
Comment (bfahr) [inline insights/parsers/rabbitmq.py]: @psachin this would be more accurate if it was ``SysconfigOptions`` instead of ``Parser``.

**Comment by bfahr** [comment]
@psachin this new spec needs to go through the spec approval process.

File: insights/parsers/rabbitmq.py (hunk 1 above)
Comment (psachin) [reply insights/parsers/rabbitmq.py]: Hi @bfahr Fixed the docstring

**Comment by psachin** [comment]
New specs proposal: **INSPEC-127**

**Pull request merged**